docs/developer/process/agent-release/pre-release.md

```diff
@@ -103,8 +103,9 @@ The main Agent release manager will increment and build a new `rc` every day a b
 
 Before each build is triggered:
 
-1. Merge any fixes that have been approved, then pull `master`
-1. Release [all changed integrations](../integration-release.md#bulk-releases) with the exception of `datadog_checks_dev`
+1. Ensure all PRs have a `category/X` label. See list of Agent Release category labels [here](https://datadoghq.atlassian.net/wiki/spaces/agent/pages/2250408413/Merging+PRs+during+RC+development#PR-categories).
```

Comment:
```suggestion
1. Ensure all PRs have a `category/X` label. For more information, see the [list of Agent Release category labels](https://datadoghq.atlassian.net/wiki/spaces/agent/pages/2250408413/Merging+PRs+during+RC+development#PR-categories).
```

Reply:
Also are we allowed to link internal sites like Atlassian on public docs?

Reply:
Good point, I'll leave the link out since the `category` labels can be seen on github. I'll add a link to that page to our internal Agent Releases confluence page.